src/transformers/models/roberta/tokenization_roberta_fast.py

```diff
@@ -170,9 +173,37 @@ def __init__(
             pad_token=pad_token,
             mask_token=mask_token,
             add_prefix_space=add_prefix_space,
+            trim_offsets=trim_offsets,
             **kwargs,
         )
 
+        # the pre_tokenizer is already updated in the GPT2TokenizerFast `__init__`
+        tokenizer_component = "post_processor"
+        tokenizer_component_instance = getattr(self.backend_tokenizer, tokenizer_component, None)
+        if tokenizer_component_instance:
+            state = json.loads(tokenizer_component_instance.__getstate__())
+
+            # The lists 'sep' and 'cls' must be cased in tuples for the object `post_processor_class`
+            if "sep" in state:
+                state["sep"] = tuple(state["sep"])
+            if "cls" in state:
+                state["cls"] = tuple(state["cls"])
```

Comment:
Aren't those potential changes to apply?

Reply:
You are right! This is also something I was asking myself, because I'm not sure what to do in case the tokens specified in the init is not part of the vocabulary (because it won't have an associated id) and I think that we have the same situation with several tokenizers (with the "special_tokens" key). Is it ok if we discuss this in another Issue/PR?

Reply:
We can definitely do it in another PR/issue.